simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java

```diff
@@ -209,4 +218,31 @@ private SignedMeterValue createSignedMeterValues(String stationId, TransactionEv
         }
         return null;
     }
+
+    DataTransferRequest createPublicKeyDataTransfer(List<Integer> connectorIds) {
```

Comment:
Does it make sense to make it per connector? Meters are per EVSE.

Moreover, in sendEichrechtMeterPublicKeys you will call this method with the same arguments for multi-EVSE configurations.

Reply:
Will change to per Evse. It makes more sense here